net/haproxy/src/opnsense/service/templates/OPNsense/HAProxy/haproxy.conf

```diff
@@ -473,7 +473,7 @@
 {%         elif action_data.type == 'tcp-request_content_reject' %}
 {%           do action_options.append('tcp-request content reject') %}
 {%         elif action_data.type == 'tcp-request_content_lua' %}
-{%           if action_data.tcp-request_content_lua|default("") != "" %}
+{%           if action_data.tcp_request_content_lua|default("") != "" %}
```

Comment:
Is this a typo? The option is actually `tcp-request_content_lua`, not `tcp_request_content_lua`.

Reply:
PyCharm indicated that there was an issue with the variable (not sure if `-` is even allowed in a variable name?). The line before has a `-` instead, but this is a string comparison. The line below is also using `action_data.tcp_request_content_lua` as a variable, so I thought this might have been wrong in the if-check. If it is indeed correct I will revert it of course.

Reply:
@mathiasaerts when the model calls it tcp_request_content_lua, it will be tcp_request_content_lua in the config as well. So in this case you are probably right.